clients/da-vinci-client/src/main/java/com/linkedin/davinci/helix/HelixParticipationService.java

```diff
@@ -158,13 +161,33 @@ private ThreadPoolExecutor initHelixStateTransitionThreadPool(int size, String t
     return helixStateTransitionThreadPool;
   }
 
+  public HelixManagerProperty buildHelixManagerProperty(VeniceServerConfig config) {
+    InstanceConstants.InstanceOperation instanceOperation;
+    if (config.getHelixJoinAsUnknown()) {
+      instanceOperation = InstanceConstants.InstanceOperation.UNKNOWN;
```

Comment:
can we start as disabled?

Reply:
From the following javadoc, when a node joins in a UNKOWN state, seems that it drops all its current assigned replicas, it sounds a good fit for a new host, but for an existing going through a deployment, is this a desired behavior or there is something else?
    
    /**
     * Behavior: Node will have all of its replicas dropped immediately and will be removed from the RoutingTableProvider.
     * Final State: Node will not hold replicas, be considered for assignment, or be populated in the RoutingTableProvider.
     */
    UNKNOWN

Reply:
Will follow up with Helix

Reply:
After talking with Helix, if an InstanceConfig already exists for a participant this change won't override it. So this UNKNOWN will only be reflected on new participants.

Reply:
Do you think it's appropriate to add that as a comment in the code to clarify the `UNKNOWN` behavior or it's a bit too LI internal?

Reply:
I think it would be a good idea to add a comment here. I'll try to make it as general as possible.